org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/commands/SurroundWithCommand.java

```diff
@@ -181,6 +181,12 @@ private List<String> getTags(DOMNode node, String prefix, int offset) {
 		Collection<CMDocument> cmDocuments = parentElement != null ? contentModelManager.findCMDocument(parentElement)
 				: contentModelManager.findCMDocument(node.getOwnerDocument(), null);
 		for (CMDocument cmDocument : cmDocuments) {
+			if (parentElement == null) {

```

Comment:
This is wrong, instead it should collect root element from all associated grammars. One second, I'll fix it.